src/webhook/mutation/pod_mutator/pod_mutator_test.go

```diff
@@ -155,7 +155,7 @@ func TestHandlePodMutation(t *testing.T) {
 		require.NotNil(t, initSecurityContext.ReadOnlyRootFilesystem)
 		assert.True(t, *initSecurityContext.ReadOnlyRootFilesystem)
 
-		assert.Nil(t, initSecurityContext.RunAsNonRoot)
+		assert.NotNil(t, initSecurityContext.RunAsNonRoot)
 
```

Comment:
As mentioned in the daily, when doing assert.True here it would pass when running the single unit test but fail when running all the package ones :clown_face:

Reply:
Fixed

Reply:
yes its veeery weird,

I think I found the problem, find 
```
func getTestPod() *corev1.Pod {
```

and make it 
```
func getTestPod() corev1.Pod {
```

Reply:
don't ask me why this is the problem 😅 

(also bunch of things will be red after the change, but fixing them is simple)

Reply:
Yes I thought about doing something similar!
I will refactor it in a new commit, thanks for checking :+1:

Reply:
@0sewa0 I tried it out and the end behavior is the same... Which makes sense because in getTestPod() the struct gets instantiated anyways whether you return it as reference or value.